src/contractTest/java/uk/gov/hmcts/reform/ccd/consumer/wa/TaskManagerDeleteTaskConsumerTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.ccd.consumer.wa;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.PactSpecVersion;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.ccd.SpringBootContractBaseTest;
+
+public class TaskManagerDeleteTaskConsumerTest extends SpringBootContractBaseTest {
+
+    private static final String WA_URL = "/task";
+    private static final String WA_DELETE_TASK = WA_URL + "/" + "delete";
```

Comment:
why don't you simply append `/delete`, rather than `"/" + "delete"`?

Reply:
Done